### PR TITLE
Added 4.7 version of OCS templates

### PR DESCRIPTION
**Pull request by Shirisha-S-Rao-1**

### CODE REVIEW

**Comment by Shirisha-S-Rao-1** [comment]
Tested the changes :

Config creation :
```
% ibmcloud sat storage config create --name ocs-remote-test --template-name ocs-remote --template-version 4.7 -p "ocs-cluster-name=testocscluster" -p "mon-size=20Gi" -p "osd-size=100Gi" -p "mon-storage-class=sat-aws-block-gold" -p "osd-storage-class=sat-aws-block-gold" -p "ibm-cos-endpoint=https://s3.us-east.cloud-object-storage.appdomain.cloud" -p "ibm-cos-location=us-east-standard" -p "ibm-cos-access-key=xxx" -p "ibm-cos-secret-key=yyy" --source-org Shirisha-S-Rao-1 --source-branch develop
Creating Satellite storage configuration...
OK
Storage configuration 'ocs-remote-test' was successfully created with ID '9443cd39-15bc-4692-a3f8-b2fb21f1dab7'.
```

Pods created:
```
% oc get pods -n openshift-storage     
NAME                                                              READY   STATUS      RESTARTS   AGE
csi-cephfsplugin-fc6lw                                            3/3     Running     0          4m35s
csi-cephfsplugin-j9cxp                                            3/3     Running     0          4m35s
csi-cephfsplugin-p4tw5                                            3/3     Running     0          4m35s
csi-cephfsplugin-provisioner-574b65c975-sb6cp                     6/6     Running     0          4m35s
csi-cephfsplugin-provisioner-574b65c975-t84c2                     6/6     Running     0          4m35s
csi-rbdplugin-2f9z5                                               3/3     Running     0          4m36s
csi-rbdplugin-6sjz9                                               3/3     Running     0          4m36s
csi-rbdplugin-dxbwg                                               3/3     Running     0          4m36s
csi-rbdplugin-provisioner-6bb4c58fdb-2q2rc                        6/6     Running     0          4m35s
csi-rbdplugin-provisioner-6bb4c58fdb-nnxnx                        6/6     Running     0          4m35s
noobaa-core-0                                                     1/1     Running     0          103s
noobaa-db-pg-0                                                    1/1     Running     0          103s
noobaa-endpoint-7c5bfc5f54-44js9                                  1/1     Running     0          27s
noobaa-operator-c94c89955-cdkp9                                   1/1     Running     0          5m13s
ocs-metrics-exporter-844cd5988b-lrzjw                             1/1     Running     0          5m12s
ocs-operator-7dcc6dc48f-tp7ks                                     1/1     Running     0          5m13s
rook-ceph-crashcollector-172.33.15.189-6b8b4fdcdf-q8nbb           1/1     Running     0          4m13s
rook-ceph-crashcollector-172.33.17.92-7b79684798-wsjtv            1/1     Running     0          2m55s
rook-ceph-crashcollector-172.33.38.136-6c677d6d5d-p9tmt           1/1     Running     0          3m31s
rook-ceph-mds-ocs-storagecluster-cephfilesystem-a-648bbd76cbrnf   2/2     Running     0          91s
rook-ceph-mds-ocs-storagecluster-cephfilesystem-b-f67445f7cm5n2   2/2     Running     0          91s
rook-ceph-mgr-a-5ddd955d66-tzn9f                                  2/2     Running     0          2m24s
rook-ceph-mon-a-5ff8c7748f-ttdwd                                  2/2     Running     0          4m20s
rook-ceph-mon-b-cc495d6c7-f6sq6                                   2/2     Running     0          3m31s
rook-ceph-mon-c-85fbc7dc57-kxmct                                  2/2     Running     0          2m55s
rook-ceph-operator-5c98f687bc-74lsz                               1/1     Running     0          5m13s
rook-ceph-osd-0-8575976b76-6pxk2                                  2/2     Running     0          2m7s
rook-ceph-osd-1-f5875f5db-l55qs                                   2/2     Running     0          118s
rook-ceph-osd-2-69b76658cb-rkp8r                                  2/2     Running     0          104s
rook-ceph-osd-prepare-ocs-deviceset-0-data-0d9s82-kgk92           0/1     Completed   0          2m23s
rook-ceph-osd-prepare-ocs-deviceset-1-data-0q994d-46sgb           0/1     Completed   0          2m22s
rook-ceph-osd-prepare-ocs-deviceset-2-data-0m9pxf-wbxkf           0/1     Completed   0          2m22s
```

Storagecluster :
```
% oc get storagecluster -n openshift-storage
NAME                 AGE   PHASE   EXTERNAL   CREATED AT             VERSION
ocs-storagecluster   6m   Ready              2021-05-31T03:38:14Z   4.7.0
```

Noobaa :
```
% oc get noobaa -n openshift-storage     
NAME     MGMT-ENDPOINTS                    S3-ENDPOINTS                      IMAGE                                                                                                            PHASE   AGE
noobaa   ["https://172.33.15.189:32177"]   ["https://172.33.38.136:30065"]   registry.redhat.io/ocs4/mcg-core-rhel8@sha256:1496a3e823db8536380e01c58e39670e9fa2cc3d15229b2edc300acc56282c8c   Ready   99s
```

**Review by akgunjal — APPROVED** [review]
LGTM

File: config-templates/redhat/ocs-remote/4.7/metadata.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    {
+      "name": "provider",
+      "value": "redhat"
+    },
+    {
+      "name": "provider-logo",
+      "value": "logo/ocs-remote.svg"
+    },
+    {
+      "name": "kube-version",
```
Comment (nkkashyap) [inline config-templates/redhat/ocs-remote/4.7/metadata.json:15]: Update key `kube-version` to `supported-k8s-versions`
Provide comma separated list of supported Kube versions like `"1.18,1.19"`

File: config-templates/redhat/ocs-remote/4.7/metadata.json [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+      "value": "1.18"
+    },
+    {
+      "name": "os-version",
+      "value": "RHEL7"
+    }
+  ]
+}
```
Comment (nkkashyap) [inline config-templates/redhat/ocs-remote/4.7/metadata.json:23]: Metadata list is incomplete, add other fields `offering-name`, `provider-url`, `support-url`,   `volume-type` 
and other pls refer to https://github.com/IBM/ibm-satellite-storage/blob/develop/config-templates/aws/aws-efs-csi-driver/1.2.2/metadata.json

File: config-templates/redhat/ocs-local/4.7/metadata.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+    {
+      "name": "kube-version",
+      "value": "1.18"
+    },
+    {
+      "name": "os-version",
+      "value": "RHEL7"
+    }
+  ]
+}
```
Comment (nkkashyap) [inline config-templates/redhat/ocs-local/4.7/metadata.json:23]: Metadata list is incomplete, add other fields `offering-name`, `provider-url`, `support-url`,   `volume-type` 
and other pls refer to https://github.com/IBM/ibm-satellite-storage/blob/develop/config-templates/aws/aws-efs-csi-driver/1.2.2/metadata.json
Rename key `kube-version` to `supported-k8s-versions`

**Review by nkkashyap — APPROVED** [review]
looks good

**Pull request merged**